src/config/subgraph.ts

```diff
@@ -1,5 +1,9 @@
 export const T_NETWORK_SUBGRAPH_URL =
   "https://api.studio.thegraph.com/query/24143/threshold/0.0.4"
 
-export const TBTC_SUBGRAPH_URL =
-  "https://api.thegraph.com/subgraphs/name/suntzu93/threshold-tbtc"
+export const T_NETWORK_SUBGRAPH_ID =
+  "5TJAMbsRwm1avUTV4CofaLT4apfQoAiNcysEit9BWr6R"
+
+export const TBTC_SUBGRAPH_ID = "DETCX5Xm6tJfctRcZAxhQB9q3aK8P4BXLbujHmzEBXYV"
+
+export const SUBGRAPH_GATEWAY_URL = `https://gateway-arbitrum.network.thegraph.com/api/${process.env.SUBGRAPH_API_KEY}/subgraphs/id/`
```

Comment:
Wondering why the subgraph gateway is `gateway-arbitrum` when tbtc is on mainnet?

Reply:
The Graph processes Layer 1 (L1) data on Arbitrum to reduce costs, enhance scalability, and improve efficiency. This enables faster querying and processing of subgraphs, making the entire curation process more cost-effective.

For more information, please refer to: https://thegraph.com/docs/en/arbitrum/l2-transfer-tools-guide/